common/test/acceptance/pages/studio/settings_certificates.py

```diff
@@ -238,15 +238,21 @@ def signatories(self):
 
     def wait_for_certificate_delete_button(self):
         """
-        Returns whether or not the certificate delete icon is present.
+        Promise to wait until certificate delete icon is available
         """
-        return self.find_css('.actions .delete').present
+        EmptyPromise(
```

Comment:
Not a must, but you could do `self.wait_for(lambda: blah-blah, "blah happened")`